docs-src/docs/reference/setup-cli.md

```diff
@@ -68,6 +68,7 @@ $ l0-setup apply [options] <instance_name>
 
 ### Options
 * `--quick` - Skips verification checks that normally run after `terraform apply` has completed
+* `--push` - Skips uploading local tfstate file to s3. (default: true)
```

Comment:
This applies here, but also is a general statement:

L0-setup `push` and `pull` is not just for `terraform.tfstate`! 
These commands sync **everything** in the `~/.layer0/<instance>` directory to/from S3. 
This is the primary way we should recommend users add custom configurations to their Layer0 instances.